packages/lit-dev-content/site/docs/ssr/client-api.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Lit SSR client API
+eleventyNavigation:
+  key: Client API
```

Comment:
Let's change this to "Client Usage" (and move declarative shadow DOM details here)